tests/test_core/test_annotations.py

```diff
@@ -0,0 +1,97 @@
+import dataclasses
+import pytest
+
+from winter.core import Component
+from winter.core import annotate
+from winter.core import annotate_class
+from winter.core import annotate_method
+from winter.core import is_component
+from winter.core.annotations import Annotations
+from winter.core.annotations import MultipleAnnotationFound
+from winter.core.annotations import NotFoundAnnotation
+
+
+@dataclasses.dataclass(frozen=True)
+class Route:
```

Comment:
Please rename `Route` to anything else, just to avoid name conflicts with winter codebase and simplify navigation in IDE.
Same for `Query`.